.expeditor/templates/verify_linux_pipeline.yml

```diff
@@ -15,8 +15,8 @@
   - label: "[@@plan@@] :linux: :habicat: Build with DO_CHECK=true"
     command:
       - bin/ci/verify-pr-build.sh @@plan@@
-    expeditor:
-      executor:
-        docker:
```

Comment:
Isn't this config still needed, to have the script execute inside the buildkite docker image?

Reply:
You are correct, I was a little over-zealous in my copy/paste/delete to get the retry